lib/DelegatedModule.js

```diff
@@ -23,6 +24,16 @@ class DelegatedModule extends Module {
 		this.delegateData = data;
 	}
 
+	contextify(context, request) {
+		return request.split("!").map(function(r) {
+			let rp = path.relative(context, r).replace(/\\/g, "/");
+			return rp.indexOf("../") !== 0 ? "./" + rp : rp;
+		}).join("!");
+	}
+
+	libIdent(options) {
+		return this.contextify(options.context, this.userRequest);
```

Comment:
When I remove contextify I get test case failures from: "ConfigTestCases dll-plugin 1-use-dll should load an harmony module from dll (star export)"

It looks like the test case has two modules './e' one in the DllModule and the other locally. They have the same name so they conflict in the bundle.